tests/unit/expire.tcl

```diff
@@ -571,6 +571,22 @@ start_server {tags {"expire"}} {
                 assert_equal [$primary pexpiretime $key] [$replica pexpiretime $key]
             }
         }
+
+        test {expired key which is created in writeable replicas should be deleted by active expiry} {
+            $primary flushall
+            $replica config set replica-read-only no
+            foreach {yes_or_no} {yes no} {
+                $replica config set appendonly $yes_or_no
```

Comment:
maybe we better call waitForBgrewriteaof here?
and also add a comment as to why we enable AOF (considering we didn't mention it in the test title).